api/envoy/extensions/filters/http/ext_proc/v3/processing_mode.proto

```diff
@@ -41,6 +41,7 @@ message ProcessingMode {
   // 1) STREAMED BodySendMode: header processing completes before body mutation comes back.
   // 2) BUFFERED_PARTIAL BodySendMode: body is buffered and could be injected in different phases.
   // 3) BUFFERED BodySendMode + SKIP HeaderSendMode: header processing (e.g., update content-length) is skipped.
+  // 4) STREAMED_AS_ARRIVE BodySendMode: Sending body regardless header response state.
```

Comment:
Naming idea: consider using STREAMED_NO_HEADER_WAIT
"Sending body regardless header response state" is a bit vague. I suggest rephrasing: "the body is streamed regardless of whether the header processing (response) has finished."

Above it says: "the content length header in three cases below ..." - this should probably be updated to four or rephrased.